__tests__/app.test.js

```diff
@@ -102,4 +104,50 @@ describe("app",()=>{
             })
         })
     })
+    describe("GET /api/articles", () => {
+        test("Status Code: 200 should return all details of articles in descending order using date /created_at", () => {
+            return request(app)
+            .get("/api/articles?sort_by=created_at")
```

Comment:
the sort by should be default, so you are correct to check that the articles are sorted as such but this should work on '/api/articles' without a query attached